emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBClient.java

```diff
@@ -210,7 +196,13 @@ public BatchWriteItemResult putBatch(String tableName, Map<String, AttributeValu
     } else {
       writeBatchList = writeBatchMap.get(tableName);
     }
-    writeBatchList.add(new WriteRequest().withPutRequest(new PutRequest().withItem(item)));
+
+    if (deletionMode) {
```

Comment:
Can you add log statements for both the cases i.e. either deleting or writing

Reply:
Done